opentreemap/treemap/js/src/lib/inlineEditForm.js

```diff
@@ -90,7 +90,7 @@ exports.init = function(options) {
             $("table[data-udf-id] .placeholder").css('display', 'none');
         },
 
-        getDataToSave = function() {
+        getDataToSave = options.getDataToSave || function() {
```

Comment:
Can you use `options.onSaveBefore` rather than a new mechanism?
* Here's where it's [implemented](https://github.com/OpenTreeMap/otm-core/blob/develop/opentreemap/treemap/js/src/lib/inlineEditForm.js#L187)
* Here's [an example](https://github.com/OpenTreeMap/otm-addons/blob/develop/cloud_management/js/src/plan.js#L45) of using it